scripts/lib/CIME/utils.py

```diff
@@ -254,6 +254,20 @@ def get_cime_root(case=None):
     logger.debug( "CIMEROOT is " + cimeroot)
     return cimeroot
 
+def get_src_root(case=None):
```

Comment:
Thanks for the cleanup, @fischer-ncar . One more change: the optional case argument should be removed now, right?

Reply:
Yes.  I've removed it.